common/shlibs

```diff
@@ -3445,3 +3445,5 @@ libbitcoin-database.so.0 libbitcoin-database-3.5.0_1
 libbitcoin-blockchain.so.0 libbitcoin-blockchain-3.5.0_1
 libbitcoin-node.so.0 libbitcoin-node-3.5.0_1
 libbitcoin-server.so.0 libbitcoin-server-3.5.0_1
+liboblibs.so.0.0 oblibs-v0.0.1.1_1
+ 
```

Comment:
Why the trailing newline here?

Reply:
https://github.com/void-linux/void-packages/pull/10813#discussion_r274401665

Reply:
You added 2 newlines

Reply:
OK ;)